tests/tracking/test_llm_utils.py

```diff
@@ -0,0 +1,42 @@
+import mlflow
+from urllib.parse import urlparse
+
+
+def test_llm_predictions_logging(tmpdir):
```

Comment:
Can we add tests for invalid inputs?

- Empty `inputs`
- `inputs`, `outputs`, and `prompots` have different legnth